bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1048,4 +1048,4 @@ spec:
   provider:
     name: Argo CD Community
   replaces: argocd-operator.v0.1.0
```

Comment:
it should be replacing `argocd-operator-v0.2.0` ?

Reply:
Thanks, I missed that.